src/EasyAbp.BookingService.Domain/EasyAbp/BookingService/AssetOccupancies/AssetOccupancyEventsHandler.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Volo.Abp.DependencyInjection;
+using Volo.Abp.EventBus.Distributed;
+using Volo.Abp.MultiTenancy;
+using Volo.Abp.ObjectExtending;
+using Volo.Abp.Uow;
+
+namespace EasyAbp.BookingService.AssetOccupancies;
```

Comment:
I found this empty class `AssetOccupancyCreatingCacheItem` in folder `AssetOccupancies`, and no reference to it.